_pages/fsa/features/fsa-quote-comparison.md

```diff
@@ -0,0 +1,213 @@
+---
+title: FSA Quote Comparison
+---
+
+**Note**: This feature is introduced with version 4.0 of the FSA Spartacus libraries.
+
+The Quote Comparison feature allows insurance customers to compare quotes of the same category. 
+
+***
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+***
+
+## Overview
+
+The main idea of this feature is to enable customers to see the differences between quotes before deciding which to choose. 
+In the default implementation, the customer can compare up to three quotes from the same product category, but this number is configurable. 
+After comparing quotes and choosing one, the customer proceeds with the checkout process for the chosen quote.
+
+## Selecting Quotes for Comparison
+
+The customer can start quote comparison from two pages:
+- the **Quote & Applications** page of the **My Account** area
+- the **Quote Details** page
+
+### Selecting Quotes from Quote & Applications Page
+
+On the Quotes & Applications page, there are three main actions that the customer can do: 
+
+1. **Filter Quotes** - Initially all quotes are visible. Filtering helps customers to quickly find the ones needed for comparison. Customers can filter quotes by category (Auto, Travel, Renewal Quotes, etc.).
+2. **Compare Quotes** - Disabled until the customer selects two quotes for comparison. This action redirects the customer to the Quote Comparison page.
+3. **Clear All** - Disabled until the customer selects at least one quote. This action resets all selected quotes.
+
+Below is an illustration of the default state (disabled) and the active state (enabled) for all three actions.
+
+![CTAs - Default State]({{ site.baseurl }}/assets/images/fsa/quote-comparison/CTAs-default-state.png)
+![CTAs - Active State]({{ site.baseurl }}/assets/images/fsa/quote-comparison/CTAs-active-state.png)
+
+Each quote card contains a checkbox named 'Select/Deselect', which allows the user to choose quotes. 
+Selecting one card of a specific category disables checkboxes on quotes from other categories. 
+Also, selecting three cards from the same category disables all other cards from that category.
+
+![Disabled State of the Checkbox]({{ site.baseurl }}/assets/images/fsa/quote-comparison/comparing-quotes-4th-disabled.png)
+
+Next to each checkbox title, there is an info tooltip, with the purpose to inform the user about the required behavior for quote comparison.
+
+![Checkbox Tooltip Info]({{ site.baseurl }}/assets/images/fsa/quote-comparison/checkbox-tooltip.png)
+
+### Selecting Quotes from Quote Details Page
+
+As said, the customer can initiate the quote comparison from the Quote Details page as well.
+
+When the user clicks the **DETAILS** button on the quote card from the Quotes & Applications page, they are presented with an overview of the relevant information for the chosen quote. 
```

Comment:
Is it maybe better to say `By clicking on the **DETAILS** button on the quote card from the Quotes & Applications page, user is presented with an overview of the relevant information for the chosen quote. `? With that we would avoid this `they` referring to user. Or perhaps use `he/she` when referring to user instead of `they`. Or `they` is used on purpose to avoid genders altogether ?

Reply:
Thanks for noticing this :) Using ''they'' is not recommended by SAP (although I prefer it, hence I oversaw it here :)). If absolutely necessary, ''he'' or ''she'' should be used (so not "'he/she", "he or she", but either ''he'' or ''she''). I agree with your proposal to avoid mentioning the user's gender, so I will rephrase the sentence.